msal/src/Microsoft.Identity.Client/Internal/Requests/InteractiveRequest.cs

```diff
@@ -156,7 +154,7 @@ private Uri CreateAuthorizationUri(bool addVerifier = false, bool addState = fal
                 _state = Guid.NewGuid().ToString();
                 requestParameters[OAuth2Parameter.State] = _state;
             }
-            
```

Comment:
>     [](start = 8, length = 4)

very minor: remove one of the empty lines

Reply:
done